types/src/transaction/change_set.rs

```diff
@@ -6,8 +6,8 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Hash, Eq, PartialEq, Serialize, Deserialize)]
 pub struct ChangeSet {
-    write_set: WriteSet,
-    events: Vec<ContractEvent>,
+    pub write_set: WriteSet,
```

Comment:
why?

Reply:
k. I'll be a bit more diligent on the call sites. :P